src/adapter/src/session/vars.rs

```diff
@@ -166,9 +166,9 @@ const TIMEZONE: ServerVar<TimeZone> = ServerVar {
     description: "Sets the time zone for displaying and interpreting time stamps (PostgreSQL).",
 };
 
-const TRANSACTION_ISOLATION: ServerVar<str> = ServerVar {
+const TRANSACTION_ISOLATION: ServerVar<IsolationLevel> = ServerVar {
     name: UncasedStr::new("transaction_isolation"),
-    value: "serializable",
+    value: &IsolationLevel::Serializable,
```

Comment:
Strict should be the default. Users can opt into "fast but unsafe", not opt in to "slow but correct".